configs/components/nssm.rb

```diff
@@ -1,14 +1,19 @@
 component "nssm" do |pkg, settings, platform|
-  pkg.version "2.24"
-  pkg.md5sum "b2edd0e4a7a7be9d157c0da0ef65b1bc"
-  pkg.url "https://nssm.cc/release/nssm-#{pkg.get_version}.zip"
+  pkg.load_from_json("configs/components/nssm.json")
 
   # Because we're unpacking a zip archive, we need to set the path to the executable.
   # We don't automatically have this set on windows, unfortunately. We need to set the
   # path to have access to 7za.
-  pkg.environment "PATH" => "/cygdrive/c/ProgramData/chocolatey/tools:$$PATH" if platform.is_windows?
+  # pkg.environment "PATH" => "/cygdrive/c/ProgramData/chocolatey/tools:$$PATH" if platform.is_windows?
 
-  win_arch = platform.architecture == "x64" ? "win64" : "win32"
+  win_arch = platform.architecture == "x64" ? "x64" : "Win32"
 
-  pkg.install_file "nssm-2.24/#{win_arch}/nssm.exe", "#{settings[:service_dir]}/nssm.exe"
+  pkg.install do
+    [
+      "/cygdrive/c/Program\\ files\\ \\(x86\\)/Microsoft\\ Visual\\ Studio/2017/BuildTools/Common7/Tools/vsdevcmd.bat",
```

Comment:
Yeah, the way this is set up will do that.

When vanagon goes to actually execute the commands in the pkg.install array, it will do so by joining each command with a `&&` so all commands in this array will be in the same process.